core/Makefile

```diff
@@ -0,0 +1,82 @@
+ifeq (/,${HOME})
+GOLANGCI_LINT_CACHE=/tmp/golangci-lint-cache/
+else
+GOLANGCI_LINT_CACHE=${HOME}/.cache/golangci-lint
+endif
+GOLANGCI_LINT ?= GOLANGCI_LINT_CACHE=$(GOLANGCI_LINT_CACHE) go run github.com/golangci/golangci-lint/cmd/golangci-lint
+
+include tools/tools.mk
+
+init: check-gopath go-mod-download install-tools
+	@echo Installing git hooks
+	@find .git/hooks -type l -exec rm {} \; && find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;
```

Comment:
This was in the original makefile. In principal this can be made entirely opt in as well under an `install-hooks/uninstall-hooks` make directive. That way we can leave it to personal preference.